packages/interactivity/src/utils.ts

```diff
@@ -36,12 +43,20 @@ const afterNextFrame = ( callback ) => {
 	} );
 };
 
-// Using the mangled properties:
-// this.c: this._callback
-// this.x: this._compute
-// https://github.com/preactjs/signals/blob/main/mangle.json
-function createFlusher( compute, notify ) {
-	let flush;
+/**
+ * Creates a Flusher object that can be used to flush computed values and notify listeners.
+ *
+ * Using the mangled properties:
+ * this.c: this._callback
+ * this.x: this._compute
+ * https://github.com/preactjs/signals/blob/main/mangle.json
+ *
+ * @param compute The function that computes the value to be flushed.
+ * @param notify  The function that notifies listeners when the value is flushed.
+ * @return The Flusher object with `flush` and `dispose` properties.
+ */
+function createFlusher( compute: Function, notify: () => void ): Flusher {
```

Comment:
I think this compute type is correct and we can let the object be inferred:

```suggestion
function createFlusher( compute: () => unknown, notify: () => void ) {
```